src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/BookingTest.kt

```diff
@@ -540,6 +540,88 @@ class BookingTest : IntegrationTestBase() {
 
   @Test
   fun `Create Temporary Accommodation Booking returns OK with correct body`() {
+    `Given a User`(roles = listOf(UserRole.CAS3_ASSESSOR)) { userEntity, jwt ->
+      `Given an Offender` { offenderDetails, inmateDetails ->
+        val premises = temporaryAccommodationPremisesEntityFactory.produceAndPersist {
```

Comment:
I wonder if there's a small opportunity to move this property+bed test set up into a helper function given it's not the focus of the test. That might then be reusable and make this and other tests like it more concise.

Reply:
I see now that this pattern of placing all set up in the test seems to be widely adopted already so any switch away should be more a coordinated effort.

Reply:
I can put a ticket in the backlog to create a `Given a Premises`/`Given a Bed` helper function and make this switch?

Reply:
It sounds good to me if you also think this type of thing would be of value.